src/CSS/ExperimentTwo/ModalStyle.css

```diff
@@ -0,0 +1,31 @@
+.CloseLayout {
```

Comment:
you should pick either `kebab-case` or `camelCase` and stick to it

Reply:
Thanks for the spot. I did not knew that its called `kebab-case`. I wonder why did they choose this name 😅